charts/internal/shoot-storageclasses/templates/storageclasses.yaml

```
@@ -18,4 +18,13 @@ metadata:
 allowVolumeExpansion: true
 provisioner: ebs.csi.aws.com
 volumeBindingMode: WaitForFirstConsumer
+---
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  name: gardener-default
```

Sorry for the back and forth, see https://github.com/gardener/gardener-extension-provider-aws/pull/146#discussion_r459248601
```suggestion
  name: default
```

---

Sorry for the back and forth @runzexia :)

---

it's ok for me😊